_fs/root-skel/etc/rc.d/sshd.sh

```diff
@@ -2,7 +2,7 @@
 
 . /etc/rc.subr
 
-DROPBEAR=/sbin/dropbear
+DROPBEAR="/usr/bin/dropbearmulti dropbear"
```

Comment:
```suggestion
DROPBEAR="/usr/sbin/dropbear"
```

should be enough:
![image](https://github.com/phoenix-rtos/phoenix-rtos-project/assets/3939983/af45f935-ca88-4e97-9234-8c67edc3fc16)